1.1/MetadataGranularities/README.md

```diff
@@ -0,0 +1,200 @@
+# Metadata Granularities
+
+This sample demonstrates the use of [Metadata in 3D Tiles 1.1](https://github.com/CesiumGS/3d-tiles/blob/b7fa517e88149942bc6166b11944bac67565369e/specification/README.md#metadata) on different levels of granularity. The sample consists of a tileset with 4 tiles where each tile has 5 contents, and the contents are assigned to two different groups. Metadata is assigned to the tileset, the tiles, each content, and to the groups.
```

Comment:
Should this link point to the `draft-1.1` branch instead? We'll just need to update it when `draft-1.1` is merged.